cli/tests/unit/timers_test.ts

```diff
@@ -178,7 +178,7 @@ unitTest(async function timeoutCallbackThis(): Promise<void> {
   const { promise, resolve } = deferred();
   const obj = {
     foo(): void {
-      assertEquals(this, window);
+      assertEquals<unknown>(this, window);
```

Comment:
This is the only case where casting to `unknown` is necessary under normal use cases.